system_test/helpers/aest_docker.erl

```diff
@@ -281,6 +318,18 @@ get_service_address(int_ws, NodeState) ->
     #{local_ports := #{int_ws := Port}} = NodeState,
     format("ws://localhost:~w/", [Port]).
 
+extract_archive(#{container_id := ID, hostname := Name} = NodeState, Path, Archive) ->
+    ok = aest_docker_api:extract_archive(ID, Path, Archive),
+    log(NodeState, "Extracted archive of size ~p in container ~p [~s] at path ~p", [byte_size(Archive), Name, ID, Path]),
+    NodeState.
+
+run_cmd_in_node_dir(#{container_id := ID, hostname := Name} = NodeState, Cmd) ->
```

Comment:
We should probably add a comment that this is supposed to be changed when `aest_docker_api:exec` supports returning the command output, right ?